### PR TITLE
Angelod/linux arm64 edge agent image

### PR DESCRIPTION
**Pull request by aribeironovaes**
This is 1 in a set of PRs to start having our ARM64 images on Linux.

### CODE REVIEW

File: edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Microsoft.Azure.Devices.Edge.Agent.Service.csproj [hunk 1]
```diff
@@ -1,11 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition="'$(Platform)' != 'ARM64'">
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)|$(Platform)' == 'Unix|ARM64'">
```
Comment (myagley) [inline edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Microsoft.Azure.Devices.Edge.Agent.Service.csproj]: Can we base this on a build time variable? For example, DotNetCore30?

File: edge-agent/docker/manifest.yaml.template [hunk 2]
```diff
@@ -6,6 +6,11 @@ manifests:
     platform:
       architecture: amd64
       os: linux
+  -
+    image: __REGISTRY__/__NAMESPACE__/azureiotedge-agent:__VERSION__-linux-arm64v8
+    platform:
+      architecture: aarch64
+      os: linux
```
Comment (myagley) [inline edge-agent/docker/manifest.yaml.template]: This needs to come in with the change to update the build images yaml updates.

File: edge-agent/docker/linux/arm64v8/base/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN apt-get update && \
+    apt-get install -y libsnappy1v5 && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /lib/aarch64-linux-gnu/libdl.so.2 /usr/lib/aarch64-linux-gnu/libdl.so.2 && \
+    ln -s /lib/aarch64-linux-gnu/libdl.so.2 /lib/aarch64-linux-gnu/libdl.so && \
+    ln -s /lib/aarch64-linux-gnu/libdl.so.2 /usr/lib/aarch64-linux-gnu/libdl.so && \
+    ln -s /lib/aarch64-linux-gnu/libc.so.6 /usr/lib/aarch64-linux-gnu/libc.so.6
+
+COPY  librocksdb.so /usr/lib
```
Comment (darobs) [inline edge-agent/docker/linux/arm64v8/base/Dockerfile]: DO we have the RocksDB build documented?

Comment (aribeironovaes) [reply edge-agent/docker/linux/arm64v8/base/Dockerfile]: Yes, but not here on github. It's documented under RocksDB github.

**Pull request merged**